source/Octopus.Server.Client/Model/Accounts/GenericOidcAccountResource.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel.DataAnnotations;
+using Octopus.Client.Extensibility.Attributes;
+
+namespace Octopus.Client.Model.Accounts
+{
+    public class GenericOidcAccountResource : AccountResource
+    {
+        public override AccountType AccountType => AccountType.GenericOidcAccount;
+
+        [Trim]
+        [Writeable]
+        public string Audience { get; set; }
+
+        [Writeable]
+        public string[] DeploymentSubjectKeys { get; set; }
+
+        [Writeable]
+        public string[] HealthCheckSubjectKeys { get; set; }
+
+        [Writeable]
+        public string[] AccountTestSubjectKeys { get; set; }
```

Comment:
This is going to be confusing for users, I know we excluded it on the UI but this allows the user to set something that has no discernable effect